src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -94,14 +94,19 @@ private static IOpenTelemetryBuilder ConfigureTracing(this IOpenTelemetryBuilder
                 {
                     o.EnrichWithHttpResponse = (activity, httpResponse) =>
                     {
-                        if (Activity.Current != null)
+                        if (Activity.Current is not null)
```

Comment:
I personally like doing: 
```suggestion
                        if (Activity.Current is { } activity)
```

and then using `activity` throughout the `if` block. But what you have is also completely valid, just a style choice.